docs/wiki/EL/el-clients.md

```diff
@@ -1,60 +1,98 @@
+- [Pruning](https://erigon.gitbook.io/erigon/basic-usage/usage/type-of-node#full-node-or-pruned-node)
 
 ### Geth
+As the official Go implementation of Ethereum, Geth (Go Ethereum) enjoys widespread adoption among developers and users alike.
+It supports various node types (full, light, archive) and is renowned for its extensive toolset and community support.
+Geth's flexibility in deployment—through package managers, Docker containers, or manual setup—ensures its versatility in diverse blockchain environments.
 
-Go-ethereum, developed by a team in EF
+Noteworthy Features:
+- [Pruning](https://geth.ethereum.org/docs/fundamentals/pruning)
+- [Custom EVM Tracer](https://geth.ethereum.org/docs/developers/evm-tracing/custom-tracer)
+- [Monitoring Dashboards](https://geth.ethereum.org/docs/monitoring/dashboards)
 
 ### Nethermind
+Written in C# .NET, Nethermind is designed for stability and integration with existing tech infrastructures.
+It offers optimized virtual machine performance, comprehensive analytics support via Prometheus/Grafana dashboards, and robust security features. 
+Nethermind is suitable for both private Ethereum networks and decentralized application (dApp) development, emphasizing data integrity and scalability.
 
-Developed in .NET
-
-## Clients in development
+Noteworthy Features:
+- [Private Networks](https://docs.nethermind.io/fundamentals/private-networks)
+- [Prometheus and Grafana](https://docs.nethermind.io/monitoring/metrics/grafana-and-prometheus)
 
 ### Reth
+Reth (Rust Ethereum) is a modular and efficient Ethereum client designed for user-friendliness and high performance.
+It emphasizes community-driven development and is suitable for robust production environments.
 
-Developed by Paradigm, recently considered stable
+Noteworthy Features:
+- [Revm](https://bluealloy.github.io/revm/)
+- [Prometheus and Grafana](https://reth.rs/run/observability.html)
 
-### Nimbus EL 
-
-Execution client developed by Nimbus team as an addition to their Nim CL work.
+### Nimbus
+Nimbus focuses on efficiency and security as an ultra-lightweight Ethereum execution layer client.
+It minimizes resource consumption while supporting Ethereum's execution layer functionalities and integrating with Fluffy (a Portal Network light client).
+Nimbus offers enhanced memory savings and state synchronization mechanisms, ideal for resource-constrained environments.
 
 ### Silkworm
-
-Modular C++ implementation by Erigon team. Also called Erigon++.
+Silkworm is a C++ implementation of the Ethereum Execution Layer protocol, aiming to be the fastest Ethereum client.
+It integrates the libmdbx database engine and emphasizes scalability and performance optimizations within the Erigon project (known as Erigon++).
 
 ### JS Client
+The TypeScript-based JavaScript client by Paradigm, part of the EthereumJS monorepo, offers flexibility and ease of integration for Ethereum's execution layer functionalities.
```

Comment:
js client is not from paradigm lol. At least afaik? It was always developed by EF, led by Holger.. 
The highlight is that they implement EIPs very soon before hardfork and give people opportunity to test stuff